tests/org.eclipse.n4js.packagejson.xpect.ui.tests/xpect/hyperlink/package.json.xt

```diff
@@ -36,13 +36,13 @@
 
 {
 	"name": "P",
-	// XPECT hyperlinks at 'ma<|>in' --> file:/.../src/main.n4js
+	// XPECT hyperlinks at '"ma<|>in"' --> file:/.../src/main.n4js
 	"main": "src/main.n4js",
 	"n4js": {
 		"projectType": "definition",
 		// XPECT hyperlinks at 'li<|>b1' --> platform:/resource/lib1/package.json
 		"definesPackage": "lib1",
-		// XPECT hyperlinks at 'ma<|>in' --> platform:/resource/P/src/main.n4js
+		// XPECT hyperlinks at '"ma<|>inModule"' --> platform:/resource/P/src/main.n4js
```

Comment:
not sure if "main" was intended to be actually the right side of the value pair.

Reply:
I wondered about this, too. So I compared the error messages from a flaky run and thought it was about the LHS.